src/_guides/openapi/specification/v3.1/understanding-structure/basic-structure.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Basic Structure
+authors: phil
+excerpt: Learn your way around the OpenAPI description format for HTTP APIs by learning the basic structure.
+date: 2024-07-02
+---
+
+The OpenAPI Specification (OAS) is an "API Description Format", providing a standard format for describing REST APIs, making it easier to design, document, and consume them.
+
+In this tutorial we'll explore the structure of an OpenAPI document, focusing on the main sections and important elements, so that you can get a feel for where everything is, without having to go through the whole OpenAPI Specification yourself, as that's meant more for tooling developers than users like yourself.
+
+## The OpenAPI Document
+
+An OpenAPI document is typically written in either YAML or JSON format, usually called `openapi.yaml` or `openapi.json`, but it could have any name.
+
+It consists of several key sections that describe the API's endpoints, requests, responses, data models, and more. Here is an outline of the main sections:
+
+1. **OpenAPI Object**
+2. **Info Object**
+3. **Servers Object**
+4. **Paths Object**
+5. **Components Object**
+6. **Security Object**
+7. **Webhooks Object**
+8. **Tags Object**
+
+Let's look at each of these objects to see what's going on.
+
+### 1. OpenAPI Object
+
+The root of the OpenAPI document is the `openapi` object, which specifies the version of the OpenAPI Specification being used. For example:
+
+```yaml
+openapi: 3.1.1
+```
+
+This is required, so that tooling knows which version you are working with. The `3.1` part is the important bit, and the patch number doesn't really matter as those "patch" versions only add clarifications to the specification and never change meaning, but it's helpful to know what version somebody was reading when they wrote the OpenAPI.
+
+### 2. Info Object
+
+The `info` object provides general metadata about the API, such as the title, version, description, and contact information.
```

Comment:
Should we mention only the `title` and the `version` is required?

Also, I think we need to clarify what the `version` field refers to. A lot of people think it's the API's version, however from the OpenAPI spec, it says it should be the **document version**:

> The version of the OpenAPI document (which is distinct from the [OpenAPI Specification version](https://spec.openapis.org/oas/v3.1.0#oasVersion) or the API implementation version).